benchmark/solkz/compositional_field/CMakeLists.txt

```diff
@@ -33,7 +33,16 @@ ENDIF ()
 DEAL_II_INITIALIZE_CACHED_VARIABLES()
 
 SET(TARGET "solkz_compositional_field")
+SET(TARGET_DEPENDENCY "solkz")
+
 PROJECT(${TARGET})
 
+INCLUDE_DIRECTORIES("..")
+
 ADD_LIBRARY(${TARGET} SHARED solkz_compositional_field.cc)
-ASPECT_SETUP_PLUGIN(${TARGET})	
+ADD_LIBRARY(${TARGET_DEPENDENCY} SHARED ../solkz.cc)
```

Comment:
Sure. I tried inserting `MESSAGE("This is ${ASPECT_SOURCE_DIR}")` to CMakeLists.txt and get the following output

```
hlokavar@ymir:~/runs/solkz_direct/test/compositional_field$ !cma
cmake -DAspect_DIR=/home/hlokavar/build/aspect/master .
This is
-- Setting up plugin:
--   name <solkz_compositional_field>
--   using ASPECT_DIR /home/hlokavar/build/aspect/master
--   in Release mode
-- Setting up plugin:
--   name <solkz>
--   using ASPECT_DIR /home/hlokavar/build/aspect/master
--   in Release mode
-- Configuring done
-- Generating done
-- Build files have been written to: /home/hlokavar/runs/solkz_direct/test/compositional_field
hlokavar@ymir:~/runs/solkz_direct/test/compositional_field$
```
After which, ran

```
hlokavar@ymir:~/runs/solkz_direct/test/compositional_field$ cmake -DAspect_DIR=/home/hlokavar/build/aspect/master -DASPECT_SOURCE_DIR=`pwd` .
This is /home/hlokavar/runs/solkz_direct/test/compositional_field
-- Setting up plugin:
--   name <solkz_compositional_field>
--   using ASPECT_DIR /home/hlokavar/build/aspect/master
--   in Release mode
-- Setting up plugin:
--   name <solkz>
--   using ASPECT_DIR /home/hlokavar/build/aspect/master
--   in Release mode
-- Configuring done
-- Generating done
-- Build files have been written to: /home/hlokavar/runs/solkz_direct/test/compositional_field
hlokavar@ymir:~/runs/solkz_direct/test/compositional_field$ 
```
Looks like this variable isn;t defined by default.